diff --git a/node_modules/cliui/build/index.cjs b/node_modules/cliui/build/index.cjs
index 82126b6..8bc6ef1 100644
--- a/node_modules/cliui/build/index.cjs
+++ b/node_modules/cliui/build/index.cjs
@@ -290,7 +290,7 @@ function cliui(opts, _mixin) {
 // Bootstrap cliui with CommonJS dependencies:
 const stringWidth = require('string-width');
 const stripAnsi = require('strip-ansi');
-const wrap = require('wrap-ansi');
+const wrap = require('wrap-ansi-cjs');
 function ui(opts) {
     return cliui(opts, {
         stringWidth,
