diff --git a/node_modules/eslint-mdx/lib/parser.js b/node_modules/eslint-mdx/lib/parser.js
index 88e9c2e..dc94af5 100644
--- a/node_modules/eslint-mdx/lib/parser.js
+++ b/node_modules/eslint-mdx/lib/parser.js
@@ -6,7 +6,7 @@ const path_1 = tslib_1.__importDefault(require("path"));
 const helpers_1 = require("./helpers");
 const processor_1 = require("./processor");
 const traverse_1 = require("./traverse");
-exports.AST_PROPS = ['body', 'comments'];
+exports.AST_PROPS = [];
 exports.DEFAULT_EXTENSIONS = ['.mdx'];
 exports.MARKDOWN_EXTENSIONS = ['.md'];
 class Parser {
